server/api/support.py

```diff
@@ -133,6 +141,68 @@ def list_organizations():
     )
 
 
+@api.route("/support/organizations/users", methods=["GET"])
+@restrict_access_support
+def list_users_by_organization():
+    string_io = io.StringIO()
+    csv_writer = csv.writer(string_io)
+    headers = ["Organization Name", "Audit Name", "Role", "Email", "Jurisdiction Name"]
+    csv_writer.writerow(headers)
+
+    elections_with_users = (
+        Election.query.filter(
+            Election.deleted_at.is_(None),
+            Election.created_at > datetime.now(timezone.utc) - timedelta(weeks=12),
+        )
+        .options(
+            subqueryload(Election.jurisdictions)
+            # pylint: disable=no-member
+            .subqueryload(Jurisdiction.jurisdiction_administrations).subqueryload(
+                JurisdictionAdministration.user
+            ),
+            subqueryload(Election.organization)
+            # pylint: disable=no-member
+            .subqueryload(Organization.audit_administrations).subqueryload(
+                AuditAdministration.user
+            ),
+        )
+        .all()
+    )
+
+    for election in elections_with_users:
+        if is_audit_complete(get_current_round(election)):
```

Comment:
Felt risky to encode this logic in the query when it might change. Of the 9 audits found, here are the runtimes for this check (which involves 1 db query):

0.0057 seconds
0.0015 seconds
0.0016 seconds
0.0037 seconds
0.0016 seconds
0.0050 seconds
0.0015 seconds
0.0016 seconds
0.0014 seconds

Reply:
What do you mean when you say `it might change`?

Reply:
Also, just checking - how come we only want completed audits and not active audits?

Reply:
The use case for this tool is to reach out for feedback from jurisdiction managers and audit admins after audits have completed

Reply:
By "it might change" I mean, as with any function, the logic that determines whether an audit is complete might be updated in the future. I mention this because I don't like doing one db call per election in post-processing, but the alternative is to reimplement the logic in the query above. In the event that `is_audit_complete` is updated later, the query in this endpoint will behave differently than expected.

Reply:
Ah I see, right. That makes sense to me